### ISSUE TITLE
preserveWhitespace not used in Utility.serialize?

### ISSUE THREAD

**Issue by olafos** [scala/scala-xml#185]
I'm wondering what is the expected result of serializing node using Utility.serialize with preserveWhitespace = false. I've analyzed the code and can't see it used anywhere.

**Comment by ashawley** [comment]
Yeah, it was added in 31b4fe10 and e18538b but seems it was a library-wide change that was aspirational but never completed.

**Comment by timzaak** [comment]
is there any new plan about this issue?

**Comment by ashawley** [comment]
I presume it's a change that could be made, but I haven't looked in to it.  Things often look easier to do than they are, but anyone is welcome to investigate and see if it could be fixed.  We'd be willing to review the fix and see if it can be released.
